java/example_code/cloudmap/src/test/java/aws/example/cloudmap/AppTest.java

```diff
@@ -0,0 +1,38 @@
+package aws.example.cloudmap;
+
+import junit.framework.Test;
+import junit.framework.TestCase;
+import junit.framework.TestSuite;
+
+/**
+ * Unit test for simple App.
+ */
+public class AppTest 
+    extends TestCase
+{
+    /**
+     * Create the test case
+     *
+     * @param testName name of the test case
+     */
+    public AppTest( String testName )
+    {
+        super( testName );
+    }
+
+    /**
+     * @return the suite of tests being tested
+     */
+    public static Test suite()
+    {
+        return new TestSuite( AppTest.class );
+    }
+
+    /**
+     * Rigourous Test :-)
+     */
+    public void testApp()
+    {
+        assertTrue( true );
+    }
+}
```

Comment:
This test doesn't seem to be doing anything.  

Is this a default file?

If not needed, can you remove this extra file, otherwise can you add tests for the DiscoverInstanceRequest class

Reply:
done the changes